Reduced logging for unfound folders

flibbles opened the pull request.

Commit:
Reduced logging for unfound folders

ScottDugas commented:
We can update the test log4j.xml to log errors and warnings on UserIterator and FindFolders now.

ScottDugas commented:
Update the main log4j.xml to be at the warn priority, so that it won't output anything by default

Commit:
Turning down logging levels

Turned down all logging to warning on main
Turned down UserIterator and FindFolders to warning on testing

ScottDugas commented on src/test/resources/log4j.xml:
```diff
@@ -37,14 +37,14 @@
          debug and above still goes to test.log -->
     <logger name="com.ripariandata.timberwolf.exchange.UserIterator"
```
You should be able to remove both of these iterator loggers from the config file outright, since the default console appender in the tests is set to only show warn and above.

Commit:
Removed those loggers entirely

Commit:
Set logging exactly as it should be for testing
